openmc/geometry.py

```diff
@@ -73,19 +73,16 @@ def export_to_xml(self, path='geometry.xml'):
             Path to file to write. Defaults to 'geometry.xml'.
 
         """
-        # Clear OpenMC written IDs used to optimize XML generation
-        openmc.universe.WRITTEN_IDS = {}
```

Comment:
On my laptop -- exporting BEAVRS using develop branch is 7.2 sec and with this one is 9.0 sec, so it is measurably slower but still same order of magnitude.